frontend/packages/db/supabase/migrations/20250319101548_add_docs_feature.sql

```diff
@@ -0,0 +1,97 @@
+create sequence "public"."DocVersion_id_seq";
+
+create sequence "public"."Doc_id_seq";
+
+revoke delete on table "public"."WatchSchemaFilePattern" from "anon";
```

Comment:
It should have been done in this migration, but apparently We forgot to include it.

https://github.com/liam-hq/liam/blob/63512be19deafb353a4d565e61dc60b08e3a13cc/frontend/packages/db/supabase/migrations/20250317054314_add_watch_schema_file_pattern.sql